frontend/src/main/components/Nav/AppNavbar.js

```diff
@@ -46,6 +46,7 @@ export default function AppNavbar({ currentUser, systemInfo, doLogout, currentUr
                 hasRole(currentUser, "ROLE_ADMIN") && (
                   <NavDropdown title="Admin" id="appnavbar-admin-dropdown" data-testid="appnavbar-admin-dropdown" >
                     <NavDropdown.Item href="/admin/users">Users</NavDropdown.Item>
+                    <NavDropdown.Item href="/admin/loadSubjects">LoadSubjects</NavDropdown.Item>
```

Comment:
Should the menu item be one word here? This is user facing menu, so I think it would make more sense to put a space in "Load Subjects".